tests/test_masking.py

```diff
@@ -26,6 +26,28 @@ def setUp(self):
             self.imlist.append(im)
         self.stack = ImageStack(self.imlist)
 
+    def test_apply_trajectory_mask(self):
+        starting_pixel = (20,20) 
+        velocity = (50.0, -10.0)
+        radius = 5.0
+        dt = self.stack.get_single_image(1).get_obstime() - self.stack.get_single_image(0).get_obstime()
```

Comment:
Same comment about `dt` as in `masking.py`

Reply:
Refactored